bugbot/user_activity.py

```diff
@@ -77,6 +81,17 @@ def __init__(
         self.activity_limit_ts = lmdutils.get_date_ymd(self.activity_limit).timestamp()
         self.seen_limit = lmdutils.get_date(reference_date, self.absent_weeks_count * 7)
 
+        self.new_user_activity_limit = lmdutils.get_date(
+            reference_date, self.new_user_weeks_count
+        )
+        self.new_user_activity_limit_ts = lmdutils.get_date_ymd(
+            self.new_user_activity_limit
+        ).timestamp()
+        self.new_user_seen_limit = lmdutils.get_date(
+            reference_date, self.new_user_weeks_count * 7
+        )
+        self.new_user_limit = self.seen_limit = lmdutils.get_date(reference_date, 61)
```

Comment:
Why 61?

Reply:
It is 61 because that is the threshold that determines if a Bugzilla account is considered "New to Bugzilla" ([Slack message](https://mozilla.slack.com/archives/C5F7KFJ4R/p1721659019778399?thread_ts=1721657796.702269&cid=C5F7KFJ4R)). However, I'm not sure if that's what you meant by considering "new users". Is there a specific set of conditions you had in mind to determine "new users"?

Reply:
No that's OK!

Reply:
Could you add a comment to explain it?

Reply:
Will do!